data/templates/metronome/domain.tpl.cfg.lua

```diff
@@ -1,15 +1,96 @@
 VirtualHost "{{ domain }}"
+  enable = true
   ssl = {
-        key = "/etc/yunohost/certs/{{ domain }}/key.pem";
-        certificate = "/etc/yunohost/certs/{{ domain }}/crt.pem";
+    key = "/etc/yunohost/certs/{{ domain }}/key.pem";
+    certificate = "/etc/yunohost/certs/{{ domain }}/crt.pem";
   }
   authentication = "ldap2"
   ldap = {
-     hostname      = "localhost",
-     user = {
-       basedn        = "ou=users,dc=yunohost,dc=org",
-       filter        = "(&(objectClass=posixAccount)(mail=*@{{ domain }})(permission=cn=xmpp.main,ou=permission,dc=yunohost,dc=org))",
-       usernamefield = "mail",
-       namefield     = "cn",
-       },
+    hostname = "localhost",
+    user = {
+      basedn = "ou=users,dc=yunohost,dc=org",
+      filter = "(&(objectClass=posixAccount)(mail=*@{{ domain }})(permission=cn=xmpp.main,ou=permission,dc=yunohost,dc=org))",
+      usernamefield = "mail",
+      namefield = "cn",
+    },
   }
+
+
+-- Discovery items
+disco_items = {
+  { "muc.{{ domain }}" },
+  { "pubsub.{{ domain }}" },
+  { "upload.{{ domain }}" },
```

Comment:
does this line need to be "jabber.{{ domain }}" ?

Reply:
absolutely